python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -132,6 +132,22 @@ $defs:
               type: bool
               default: false
               description: Enable PTP.
+            role:
+              type: list
+              description: Role in boundary clock mode. EOS default is `dynamic`.
```

Comment:
```suggestion
              description: Role in boundary clock mode for each node. Default is `dynamic`.
```

Reply:
Updated and also added `default: dynamic`